Mathlib/Tactic/Linarith/Verification.lean

```diff
@@ -190,17 +219,9 @@ tactic, which is typically `ring`. We prove (2) by folding over the set of hypot
 def proveFalseByLinarith (transparency : TransparencyMode) (oracle : CertificateOracle)
     (discharger : TacticM Unit) : MVarId → List Expr → MetaM Expr
```

Comment:
The match syntax (`| _, [] => ...`) is now unnecessary, right? You can just have a syntax
```
def proveFalseByLinarith (transparency : TransparencyMode) (oracle : CertificateOracle)
    (discharger : TacticM Unit) (g : MVarId) (l : List Expr) : MetaM Expr := do
```
and the main body of the function.